File: scala/src/main/scala/folio/codinginterview/domain/AppConstants.scala
package folio.codinginterview.domain

/** アプリケーション定数。 */
object AppConstants {
  val cashRate: BigDecimal = BigDecimal("0.05")

  val minOperationAmount: BigDecimal = BigDecimal(10000)

  val supportedSymbols: Seq[StockSymbol] = Seq(StockSymbol.Toyopa, StockSymbol.Somy)

  val initialPortfolio: Portfolio = Portfolio(
    Seq(
      PortfolioItem(StockSymbol.Toyopa, BigDecimal("0.40")),
      PortfolioItem(StockSymbol.Somy, BigDecimal("0.60"))
    )
  )
}

File: php/src/CodingInterview/Domain/AppConstants.php
<?php

declare(strict_types=1);

namespace Folio\CodingInterview\Domain;

final class AppConstants
{
    public static function cashRate(): BigDecimal
    {
        return new BigDecimal('0.05');
    }

    public static function minOperationAmount(): BigDecimal
    {
        return new BigDecimal('10000');
    }

    /** @return StockSymbol[] */
    public static function supportedSymbols(): array
    {
        return [StockSymbol::Toyopa, StockSymbol::Somy];
    }

    public static function initialPortfolio(): Portfolio
    {
        return new Portfolio([
            new PortfolioItem(StockSymbol::Toyopa, new BigDecimal('0.40')),
            new PortfolioItem(StockSymbol::Somy, new BigDecimal('0.60')),
        ]);
    }
}
